### ISSUE TITLE
logistic error while handling folders without name

### ISSUE THREAD

**Issue by xmcp** [ht35268/bzShare#11]
If people rename a folder to an empty name, they can never change it back, and the files in it are forever unaccessible. What's more, deleting the empty-named folder will also clear it's parent folder.

**Comment by ht35268** [comment]
As you mentioned, this would contribute to the `make_nice_filename` function which should, theoretically, disallow such filenames. I will solve this later... maybe this Saturday?

**Comment by ht35268** [comment]
By the way I wonder who injected that **_______** file into my folder? It will hurt to remove the file.

Anyway I am going to update the database a while later. Never mind.

**Comment by xmcp** [comment]
you mean the `you\ncannot\nrename\nme\ncan\nyou`?

dude, deleting it is just fine, it's just unable to rename. here is the reason:

```html
<button class="btn btn-default" type="button" data-toggle="modal" data-target="#dialog-input-string-container" onclick="javascript:bzsDialogInputStringLoad('Rename file', 'you
cannot  /* from here: Uncaught SyntaxError: Invalid or unexpected token */
rename
me
can
you', '/files/operation', '4c30466b62576c7561584e30636d46306233497665573931436d4e68626d3576644170795a5735686257554b6257554b59324675436e6c7664513d3d', bzsFilesMarkRename)" ><div class="ion ion-edit"></div></button>
```

**Comment by ht35268** [comment]
I get it.

**Comment by ht35268** [comment]
b932b303d812c7693f868d4cda6cf84db2b5676f#diff-1e0756c2ef9291994c4007d09648153dR295

Fixed #11.

**Comment by ht35268** [comment]
Ah... I haven't solve the null-name issue...
